### ISSUE TITLE
ADC MCP3208

### ISSUE THREAD

**Issue by mattyhunter** [kizniche/Mycodo#1215]
Hey mate, i want to ask you, if there is any possibility to add support for ADC MCP3208. It should be very similar to MCP 3008,.

Thank you for your reply. Have a nice day!
M.

**Comment by kizniche** [comment]
I can't find a Python library for this device.

**Comment by kizniche** [comment]
I found a library that is no longer in pypi, at https://github.com/MomsFriendlyRobotCompany/mcp3208. I'm unsure if it works and I don't have this device to test. Nonetheless, I created an Input module that uses the library and should work, in commit https://github.com/kizniche/Mycodo/commit/4a1bcf89c03fe80d3d260aa22253ff111a582d45. To test the latest this module in the latest code, prior to release, you'll need to [upgrade to master](https://github.com/kizniche/Mycodo/wiki/Updating-to-Github-Master-Branch).
